SwiftLeeds/Views/About/ContributorsView.swift

```diff
@@ -34,7 +26,7 @@ struct ContributorsView: View {
             )
             Group {
                 StackedTileView(primaryText: "About", secondaryText: Strings.aboutContributor)
-                sectionHeader(text: "Contributors")
+                CommonCompo.sectionHeader(text: "Contributors")
```

Comment:
This could just be a custom view 😊

It reads nicer as SectionHeader(text) for sure

Reply:
Hmmm.. do you want me to revert back to the previous.

Reply:
I think a proper discrete view is best! 🙏🏻

Reply:
Got it